src/java/com/media/FileProcessor.java

```diff
@@ -48,7 +57,8 @@ private FileProcessor(File rawDest, File standardDest, File mobileDest, List<Str
 		
 		this.commands = commands;
 	}
-	
+
+
 	/** Assumes that there will already be an image in /uploads/raw for it to find **/
 	public static FileProcessor ImageProcessor(File rawDest, File standardDest, File mobileDest) {
 		String inputImagePath = rawDest.getAbsolutePath();
```

Comment:
For security against injection attacks, all file paths should be run through `FileUtils.safeFilename()`

Reply:
In this case MediaUploadServlet (the only current route to FileProcessor) already handles that - FileUploadField uses safeFilename to mangle incoming filenames appropriately.